flagd/pkg/service/flag-sync/handler.go

```diff
@@ -60,7 +61,8 @@ func (s syncHandler) GetMetadata(_ context.Context, _ *syncv1.GetMetadataRequest
 	*syncv1.GetMetadataResponse, error,
 ) {
 	metadata, err := structpb.NewStruct(map[string]interface{}{
-		"sources": s.mux.SourcesAsMetadata(),
+		"sources":        s.mux.SourcesAsMetadata(),
+		"context_values": s.contextValues,
```

Comment:
Instead of this property, can you expand the s.contextValues into the existing map, with the existing "sources" taking priority in case of conflicts (if there's another key called "sources" I mean)?

This field `context_values` is not specified anywhere, and currently, a few of our providers are adding all the properties here to their context, not specifically looking for this field.

Reply:
Thanks!